backend/endpoints/rooms.py

```diff
@@ -75,6 +78,75 @@ async def create_room(
         detail="unable to create room",
     )
 
+@router.patch("/org/{org_id}/rooms/{room_id}/members/{member_id}",
+    response_model=ResponseModel, 
+    status_code=status.HTTP_200_OK,
+    responses={
+        200: {"detail": {"room": "room"}},
+        404: {"detail": "room or member not found"},
+        424: {"detail": "member removal unsuccessful"},
+    },)
+async def remove_member(org_id: str, member_id: str, room_id: str, admin_id: str):
```

Comment:
As it is, `admin_id` is required. But `admin_id` should be optional with None as a default value.
```suggestion
async def remove_member(org_id: str, room_id: str, member_id: str, admin_id: Optional[str] = None):
```